### PR TITLE
[12.0] [MIG] hr_payslip_change_state

### PR DESCRIPTION
**Pull request by bjeficent**
Migration to version 12.0

### CODE REVIEW

File: hr_payslip_change_state/tests/test_hr_payslip_change_state.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# © 2019 - Eficent http://www.eficent.com/
```
Comment (cubells) [inline hr_payslip_change_state/tests/test_hr_payslip_change_state.py]: change `©` by `Copyright` please.

File: hr_payslip_change_state/wizard/hr_payslip_change_state.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# © 2019 - Eficent http://www.eficent.com/
```
Comment (cubells) [inline hr_payslip_change_state/wizard/hr_payslip_change_state.py]: change `©` by `Copyright` please.

File: hr_payslip_change_state/wizard/hr_payslip_change_state_view.xml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+<!--© 2019 - Eficent http://www.eficent.com/-->
```
Comment (cubells) [inline hr_payslip_change_state/wizard/hr_payslip_change_state_view.xml]: change `©` by `Copyright` please.

**Review by cubells — APPROVED** [review]
Minor changes but module works as expected. So I approve.

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

**Comment by pedrobaeza** [comment]
Please squash together the 2 translation files from the same person + remove the change in oca_dependencies.txt

**Comment by bjeficent** [comment]
Commits squashed and oca_dependencies removed.

File: hr_payslip_change_state/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,18 @@
+# Copyright 2019 - Eficent http://www.eficent.com/
+# Copyright 2019 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
```
Comment (nikul-serpentcs) [inline hr_payslip_change_state/__manifest__.py:4]: @bjeficent Add `website` key and URL should be `https://github.com/OCA/hr`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
MinorChange

**Comment by bjeficent** [comment]
Comments attended, thanks for reviewing.

**Review by AaronHForgeFlow — APPROVED** [review]
Tested on local :+1: Working as expected.

**Comment by AaronHForgeFlow** [comment]
@nikul-serpentcs could you update your review please?

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

**Pull request merged**